modules/client-configuration/pages/system-locking.adoc

```diff
@@ -0,0 +1,54 @@
+[[system=locking]]
+= System Locking
+
+System locks are used to prevent actions from occurring on a client.
+For example, a system lock will prevent a client from being updated or restarted.
+This is useful for clients running production software, or to prevent accidental changes.
+You can disable the system lock when you are ready to perform actions.
+
+System locks are implemented differently on traditional and Salt clients.
+
+
+
+== System Locks on Traditional Clients
+
+When a traditional client is locked, no actions can be scheduled, and a padlock icon is displayed next to the name of the client in the menu:System[System List].
```

Comment:
Remote commands can still be issued an executed even when a traditional system is locked